### DISCUSSION TITLE
cyrillic letters for FlxInputText

### DISCUSSION THREAD

**Discussion by chekisteg1986** [HaxeFlixel/flixel#2642] [Q&A]
Helllo there. I have some issue.
So, FlxInputText cant use cyrillic letters.
I investigate it, so problem in  function

private function onKeyDown(e:KeyboardEvent):Void

for every cyrillic char (russian, ukrainian) e.charCode == 0.
I cant get where is e.charCode get its value, i just want to upgrade it so i can use cyrillic for text input.

**Comment by Geokureli** [comment]
@ShaharMS was working on improving FlxInputText, but hasn't finished. You can use their [texter](https://github.com/ShaharMS/texter) lib, for now

**Reply by chekisteg1986** [reply]
> 

oh, thank you! i'll try it now

**Reply by ShaharMS** [reply]
If you encounter any issue, please contact me/make an issue on the github repo. Just like @Geokureli said, things are still a bit unfinished since the more "advanced" versions adds many things (better multiline, bidi, multilingual text...) and some things might not be as battle tested. So, yeah, report If you have any issue :)
